plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestIcebergSnowflakeCatalogConnectorSmokeTest.java

```diff
@@ -121,6 +121,33 @@ REGIONKEY NUMBER(38,0),
                 .build();
     }
 
+    @Override
+    protected void createSchema(String schemaName)
+            throws SQLException
+    {
+        server.execute(schemaName, "CREATE SCHEMA %s".formatted(schemaName));
+    }
+
+    @Override
+    protected void dropSchema(String schema)
+            throws SQLException
+    {
+        server.execute(schema, "DROP SCHEMA %s".formatted(schema));
```

Comment:
nit: We could use + operator instead of formatted method. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#prefer-string-formatting